app/models/user.rb

```diff
@@ -7,6 +7,10 @@ class User < ApplicationRecord
 
   validates_presence_of :first_name, :last_name
 
+  def full_name
+    "#{first_name} #{last_name}"
+  end
+
```

Comment:
@Ashley, If we go for making `full_name` a user attribute (which I agree with), I don't see any advantage to having a `member_full_name` in the `MembershipsApplicationHelper`. A refactoring to a `full_name` delegate on `MembershipApplication` would then seem a better fit.

But that would mean changes to some tests, views and helpers that have actually nothing to do with this feature. So I think that refactoring would probably be best done in a separate chore.